packages/build/.eslintrc.js

```diff
@@ -134,6 +140,15 @@ module.exports = {
     'react/self-closing-comp': 0,
     'react/sort-comp': 0,
     'react/jsx-wrap-multilines': 1,
+    // allowExpressions allow single expressions in a fragment eg: <>{children}</>
+    // https://github.com/jsx-eslint/eslint-plugin-react/blob/f83b38869c7fc2c6a84ef8c2639ac190b8fef74f/docs/rules/jsx-no-useless-fragment.md#allowexpressions
+    'react/jsx-no-useless-fragment': [2, { allowExpressions: true }],
+
+    // TODO: remove checking for production env after all warnings are fixed.
+    // But meanwhile, warn dev's to aid with new/touched files.
+    'react-hooks/rules-of-hooks': process.env.NODE_ENV === 'production' ? 0 : 1,
```

Comment:
As this is just a warning is it necessary to disable it for production? Won't it just add a few extra lines in the build output but have no impact on the build itself?

Reply:
it prevents the build from passing

Reply:
actually good call, i mistook pretty warning errors that failed the build as react hook warning errors